Gallery-Server/src/main/resources/application.properties

```diff
@@ -3,8 +3,9 @@
 quarkus.datasource.db-kind=h2
 quarkus.hibernate-orm.database.generation=update
 quarkus.hibernate-orm.log.sql=true
-quarkus.datasource.jdbc.url=jdbc:h2:./data/${DatabaseName}
+# quarkus.datasource.jdbc.url=jdbc:h2:./data/${DatabaseName}
```

Comment:
any need for the comment?

Reply:
fixed, just demoing something for the other group